roles/edpm_install_certs/tasks/main.yml

```diff
@@ -14,6 +14,9 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Adopt (stop tracking) certs from tripleo
+  ansible.builtin.include_tasks: adoption.yml
```

Comment:
i don't think this is something we want included every time, even for greenfield deployments. Could we include it from the `edpm_tripleo_cleanup` role instead?

Reply:
Yeah, makes sense, I moved it there.